src/AmbientSounds/Cache/SoundCache.cs

```diff
@@ -46,6 +46,13 @@ public async Task<IReadOnlyList<Sound>> GetPreinstalledSoundsAsync()
         return _preinstalled.Values as IReadOnlyList<Sound> ?? Array.Empty<Sound>();
     }
 
+    /// <inheritdoc/>
+    public async Task<Sound?> GetInstalledSoundAsync(string stringId)
+    {
+        await GetInstalledSoundsAsync();
+
+        return _installedSounds.TryGetValue(stringId, out Sound result) ? result : null;
+    }
```

Comment:
why was this moved?

Reply:
This way it's right above the version taking an array of ids, figured it made sense to keep the two close together.